### PR TITLE
Show upload only if it is possible to upload

### PR DESCRIPTION
**Pull request by pusztaienike**
The final style will be solved during the global style refactor.

### CODE REVIEW

**Comment by herflis** [comment]
I don't know if it is related or not but after login the addbutton is active and after clicking it, it opens an empty menu

image

File: apps/sensenet/src/components/AddButton.tsx [hunk 1]
```diff
@@ -173,20 +178,45 @@ export const AddButton: React.FunctionComponent<AddButtonProps> = props => {
...
+      })}>
       {!props.isOpened ? (
         <div className={classes.iconButtonWrapper}>
-          <Tooltip title={localization.addNew} placement="right">
+          {isAvailable ? (
+            <Tooltip
+              title={localization.addNew}
+              placement="right"
+              className={clsx({
+                [classes.disabled]: !isAvailable,
```
Comment (B3zo0) [inline apps/sensenet/src/components/AddButton.tsx]: In `!isAvailable` state it renders another component so this class is not needed IMO.

File: apps/sensenet/src/components/drawer/PermanentDrawer.tsx [hunk 2]
```diff
@@ -67,11 +67,26 @@ const useStyles = makeStyles((theme: Theme) => {
   })
 })
 
+const isLocalization = (pathname: string) => {
+  const res = decodeURIComponent(pathname).split('/')
```
Comment (B3zo0) [inline apps/sensenet/src/components/drawer/PermanentDrawer.tsx]: is this 2 function needed? can't we use the matchPath here as well?

File: apps/sensenet/src/components/drawer/PermanentDrawer.tsx [hunk 3]
```diff
@@ -99,7 +114,11 @@ const PermanentDrawer: React.FunctionComponent<RouteComponentProps> = props => {
           </div>
 
           {matchPath(props.location.pathname, `/:repositoryId/saved-queries`) === null ? (
-            <AddButton isOpened={opened} path={currentPath} />
+            matchPath(props.location.pathname, { path: `/`, exact: true }) === null &&
```
Comment (B3zo0) [inline apps/sensenet/src/components/drawer/PermanentDrawer.tsx]: I would like to see this refactored. Lets talk about it sometime.

File: apps/sensenet/src/components/drawer/PermanentDrawer.tsx (hunk 2 above)
Comment (pusztaienike) [reply apps/sensenet/src/components/drawer/PermanentDrawer.tsx]: Yes, we need them, cause in these cases we encode the Url datas (use-drawer-items.tsx line 83-133) We cannot get these info any other way. 
After the URL refactor, we should change this logic here as well (so it is just a temporary solution)

Comment (pusztaienike) [reply apps/sensenet/src/components/drawer/PermanentDrawer.tsx]: The perfect solution was to create new componets for localization and users and groups with their own URL

File: apps/sensenet/src/components/drawer/PermanentDrawer.tsx (hunk 3 above)
Comment (pusztaienike) [reply apps/sensenet/src/components/drawer/PermanentDrawer.tsx]: I changed the blacklist to a whitelist, so we don't need to create loop for the checks (cause it contains only 3 conditions)

File: apps/sensenet/src/components/MainRouter.tsx [hunk 4]
```diff
@@ -22,6 +22,12 @@ const EditProperties = lazy(async () => await import(/* webpackChunkName: "editP
 const DocumentViewerComponent = lazy(async () => await import(/* webpackChunkName: "DocViewer" */ './DocViewer'))
 
 const TrashComponent = lazy(async () => await import(/* webpackChunkName: "Trash" */ './trash/Trash'))
+const UsersAndGroupsComponent = lazy(
+  async () => await import(/* webpackChunkName: "Trash" */ './users-and-groups/users-and-groups'),
```
Comment (B3zo0) [inline apps/sensenet/src/components/MainRouter.tsx]:
```suggestion
  async () => await import(/* webpackChunkName: "UserAndGroup" */ './users-and-groups/users-and-groups'),
```

File: apps/sensenet/src/components/MainRouter.tsx [hunk 5]
```diff
@@ -22,6 +22,12 @@ const EditProperties = lazy(async () => await import(/* webpackChunkName: "editP
...
+)
+const LocalizationComponent = lazy(
+  async () => await import(/* webpackChunkName: "Trash" */ './localization/localization'),
```
Comment (B3zo0) [inline apps/sensenet/src/components/MainRouter.tsx]:
```suggestion
  async () => await import(/* webpackChunkName: "Localization" */ './localization/localization'),
```

**Pull request merged**